ruby_on_rails/assets_and_navigation/turbo_drive.md

```diff
@@ -1,6 +1,6 @@
 ### Introduction
 
-Rails has always promoted itself as a framework that makes building a web application easy, but in a world built around the demands of users expecting lightning fast responses as well as increasingly complex UI it wasn't easy for Rails to offer a compelling vision for how they saw a way through that complexity while sticking to core Rails philosophies. However just before the release of Rails 7 we finally got a glimpse of how the Rails team saw the future of meeting these demands. Hotwire!
+Rails has always promoted itself as a framework that makes building a web application easy. But, in a world built around the demands of users expecting lightning fast responses as well as increasingly complex UI, it wasn't easy for Rails to offer a compelling vision for how they saw a way through that complexity while sticking to core Rails philosophies. However, just before the release of Rails 7 we finally got a glimpse of how the Rails team saw the future of meeting these demands. Hotwire!
```

Comment:
The but needs to be capitalized, but typically we wouldn't put a comma after the "But". That's my only nitpick and then I would be happy to approve this PR after the comma is removed.

Reply:
You're right, that would be better. I made the change now.